packages/path_provider/path_provider_android/test/path_provider_android_test.dart

```diff
@@ -17,7 +17,16 @@ const String kExternalCachePaths = 'externalCachePaths';
 const String kExternalStoragePaths = 'externalStoragePaths';
 const String kDownloadsPath = 'downloadsPath';
 
+enum ExternalStoragePathsValueState { isEmpty, isNotEmpty }
+
 class _Api implements TestPathProviderApi {
+  _Api({
+    this.externalStoragePathsValueState =
+        ExternalStoragePathsValueState.isNotEmpty,
+  });
+
+  final ExternalStoragePathsValueState externalStoragePathsValueState;
```

Comment:
This can just be `final bool returnsExternalStoragePaths;` now, and the enum can be removed.